samples/openapi3/client/petstore/python-experimental/petstore_api/models/drawing.py

```diff
@@ -69,7 +69,7 @@ class Drawing(ModelNormal):
     validations = {
     }
 
-    additional_properties_type = None
+    additional_properties_type = (one_ofapplebanana.OneOfapplebanana,)  # noqa: E501
```

Comment:
@spacether , here this is not correctly generated, it should be a reference to the `Fruit` class defined in ./petstore_api/models/fruit.py.

Reply:
This issue could be handled in that function or it could be handled in getTypeString.

My preference is to handle it in getTypeString because that is our function that handles conversions from Schemas to python types.

How about in get_type string we add an in block that handles the case where schema is of type composed schema? If that is the case we need to convert the referenced model name into a python module style model name, so Pet should become pet.Pet with:
```
#get the model name form the reference
String modelName = ModelUtils.getSimpleRef(ref);
modelName = toModelName(modelName);
```
Are we also handling this case for:
- object type models in additionalproperties
- primitive models (non-object and non-array type) with validations and enums in additionalproperties?
  - should we handle those in a separate PR?

Reply:
> This issue could be handled in that function or it could be handled in getTypeString.
> 
> My preference is to handle it in getTypeString because that is our function that handles conversions from Schemas to python types.

Thanks for the suggestion. Done

> 
> Are we also handling this case for:
> 
> * object type models in additionalproperties
> * array type models in additionalproperties
> * primitive models (non-object and non-array type) WITHOUT validations and enums in additionalproperties? We do not produce models for these so we should return the simple type from the ref model
> * primitive models (non-object and non-array type) with validations and enums in additionalproperties? We do produce these models so we should return a type like model_simple.ModelSimple
>   
>   * should we handle those in a separate PR?

This will be handled in a separate PR.